content/lua/concepts/mathematical-library/terms/exp/exp.md

```diff
@@ -0,0 +1,44 @@
+---
+Title: 'Exp'
+Description: 'Exp is method used to calculate the exponential value of a number.'
+Subjects:
+  - 'Computer Science'
+  - 'Lua'
+  - 'Game Development'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+  - 'Math'
+  - 'Numbers'
+CatalogContent:
+  - 'learn-lua'
+  - 'paths/computer-science'
+---
+
+In Lua, the **math.exp()** function is a standard function that is part of the Lua 'math' library. The math.exp() function is used to calculate the exponential value of a number, i.e, the exponential constant 'e' (which is roughly equal to 2.718281828459045...) raised to a given value. Exponents are a very common part of mathematical operations across a wide variety of fields and disciplines such as Physics, engineering, probability and statistics, and even machine learning! This means that the math.exp() function is a very useful and widely applicable tool for Lua developers.
+
+As a side-note, the **math.log()** function in the math library returns the inverse of the math.exp() function.
+
+## Syntax
+
+As .exp() is a method that is a part of the standard Lua Math library, to use this function it must be called as 'math.exp()'
+
+```lua
+
+math.exp(mycoolvalue) -- returns the result of e raised to the power of mycoolvalue
+
+```
+
+## Example
```

Comment:
There are some things that need to be changed with this example. The code needs to be changed so the code produces some output to the console. Then, instead of a comment in the code, the output needs to be shown in a ` ```shell` block after the example like this:

````
This example results in the following output:

```shell
Output from example
```
````
We also need an introductory sentence before the example summarizing what the example is doing.